opencti-platform/opencti-front/src/private/components/cases/case_rfis/CaseRfiDeletion.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import { graphql } from 'react-relay';
+import { useNavigate } from 'react-router-dom';
+import { useFormatter } from '../../../../components/i18n';
+import Security from '../../../../utils/Security';
+import Transition from '../../../../components/Transition';
+import { KNOWLEDGE_KNUPDATE_KNDELETE } from '../../../../utils/hooks/useGranted';
+import useDeletion from '../../../../utils/hooks/useDeletion';
+import useApiMutation from '../../../../utils/hooks/useApiMutation';
+
+const caseRfiDeletionDeleteMutation = graphql`
+  mutation CaseRfiDeletionDeleteMutation($id: ID!) {
+    caseRfiDelete(id: $id)
+  }
+`;
+
+const CaseRfiDeletion = ({ id }: { id: string }) => {
+  const { t_i18n } = useFormatter();
+  const navigate = useNavigate();
+  const deleteSuccessMessage = t_i18n('', {
+    id: '... successfully deleted',
+    values: { entity_type: t_i18n('entity_Case-Rfi') },
+  });
+  const [commit] = useApiMutation(
+    caseRfiDeletionDeleteMutation,
+    undefined,
+    { successMessage: deleteSuccessMessage },
+  );
+  const handleClose = () => { };
+
+  const {
+    deleting,
+    handleOpenDelete,
+    displayDelete,
+    handleCloseDelete,
+    setDeleting,
+  } = useDeletion({ handleClose });
+
+  const submitDelete = () => {
+    setDeleting(true);
+    commit({
+      variables: {
+        id,
+      },
+      onCompleted: () => {
+        setDeleting(false);
+        handleClose();
+        navigate('/dashboard/cases/rfis');
+      },
+    });
+  };
+
+  return (
+    <React.Fragment>
```

Comment:
```suggestion
    <>
```
Not a big deal but it is to be consistent with the way things are done in the platform 🙂